modules/core/src/main/scala/lucuma/odb/api/repo/ObservationRepo.scala

```diff
@@ -154,5 +159,31 @@ object ObservationRepo {
           Tables.constraintSetObservation,
           ConstraintSetModel.ConstraintSetEvent.updated
         )
+
```

Comment:
We need to be able to unset the link from explore without knowing the currently linked constraint set. It is possible to get the current link in explore, but the final API probably won't require it, so I'd like to avoid plumbing all that in only to rip it out.